dotcom-rendering/src/web/components/SupportingContent.tsx

```diff
@@ -40,26 +41,45 @@ const directionStyles = (alignment: Alignment) => {
 	}
 };
 
+const dynamoStyles = css`
+	flex-direction: column;
+	column-gap: 5px;
+	margin: 0;
+
+	${from.tablet} {
+		margin: 0 5px 5px;
+		flex-direction: row;
+		position: absolute;
+		bottom: 0;
+	}
+`;
+
 const liStyles = css`
 	display: flex;
 	flex-direction: column;
 	flex: 1;
 	padding-top: 2px;
 	position: relative;
 	margin-top: 8px;
-	${from.phablet} {
+	${from.tablet} {
 		margin-bottom: 4px;
 	}
 `;
 
+const dynamoLiStyles = css`
+	background-color: ${transparentColour(neutral[97], 0.875)};
+	border-top: 1px solid ${news[400]};
```

Comment:
Here is a question I don't know the answer to: In @DanielCliftonGuardian's PR this border was getting the colour of the parent. I'm wondering if this is the requirement. I know we discussed this briefly yesterday and I was the one who suggested using the `news` pillar but now I have doubts. Having said that, I don't think I have seen this container being used for another pillar. Some investigation into the frontend code might be worth here if we don't have any other way to know the requirement.

Reply:
Yes, we could definitely fix forward. I did a bit of an investigation and I think it's coming from the palette. Old palettes css here: 
https://github.com/guardian/interactive-atom-container-colours/blob/master/shared/css/_dynamoPalettes.scss#L320
https://github.com/guardian/interactive-atom-container-colours/blob/master/shared/css/_variables.scss#L94 (This is for eventPalette which is what we're using as an example.

Reply:
Approving!

Reply:
I’ve made it come through via the palette, now.